hadoop-hdds/docs/content/feature/Nonrolling-Upgrade.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Non-rolling upgrades and downgrades
+summary: Steps to do a non rolling upgrade of Ozone.
+date: 2021-02-15
+author: Aravindan Vijayan 
```

Comment:
I am not sure whether we should keep `author` property here, as other docs do not have this.

Reply:
Hi @iamabug,I think `author` property is optional, because `author` property is still reserved in some Ozone documents (https://github.com/apache/ozone/tree/master/hadoop-hdds/docs/content/design).

Only personal thoughts～

Reply:
@errose28 Let's remove the author.